tests/nn/data_parallel/test_sharded_ddp_pytorch_parity.py

```diff
@@ -173,6 +181,13 @@ def closure_sharded(input_tensor=input_tensor):
                 next(ddp_model.parameters()).requires_grad = not next(ddp_model.parameters()).requires_grad
                 check_same_model_params(sharded_ddp_model, ddp_model, f"Rank: {rank} - Trainability refresh {i} broke")
 
+                if clip_grad_norm:
```

Comment:
this is extensively tested on the OSS side, but in that case the gradients are not sharded, so this bug was not caught. I added a simple variant here, checking with one norm only but with the sharded gradients